tchannel/tornado/connection.py

```diff
@@ -192,6 +195,8 @@ def on_error(future):
 
             if isinstance(exception, tornado.iostream.StreamClosedError):
                 self.close()
+            if message_future.running():
```

Comment:
Tests only cover behavior if they make assertions about that behavior. Lots of tests might be hitting this code path, but this is behavior we need to explicitly test. Are we doing that?